scripts/lib/mkcloud-libvirt.sh

```diff
@@ -103,6 +103,11 @@ function libvirt_prepare()
 
 function libvirt_setupadmin()
 {
-    ${mkcloud_lib_dir}/libvirt/admin-config $cloud $admin_node_memory $adminvcpus $emulator $admin_node_disk "$localreposdir_src" "$localreposdir_target" > /tmp/$cloud-admin.xml
+    nvramtemplate=""
```

Comment:
I understood that libvirt uses the NVRAM master from qemu.conf which is default and the "template" is used to override the master with the template needed for ovmf. I could change to not pass any template parameter for the general case and use the template file below for OVMF. Would that be better ?